README.md

```diff
@@ -1,18 +1,18 @@
 4 | Communication | <ul><li>Product/Service provides rationale of decision to user, tests of primary langauge demonstrates positve outcomes, translations begins, tests for visuals begin, countermeasures generates positive outcomes, assessment for UN Declarations commences</ul></li>
-5 | Adaptation | <ul><li>Product/Service working group embodies BIPOC, LGBTQIA2S+, Neurodiverse, People with Disabilities, Rural Residents, Socio-economic diverse, and Gender Diverse Folx.</ul></li></ul></li><ul><li>Feedback system incorporates empathy, confidentiality and Equity Impact Assessment(s), inclusive of Health, open-ended questions & vulnerability, occur frequently for continious improvement, and is delivered to working group immediately and efficiently for action. Follow-up with client includes humility and accountability.</ul></li><ul><li>Framework in place for ensuring stereotypes, implicit bias, and/or prejudice does not occur.</ul></li><ul><li>Privacy and seucrity is assured for everyone through privacy preservation tools; there is not a need for marginalized groups to have more than one personna </ul></li><ul><li>Demographic (race, gender, immigration), health and other data is used in aggravated form, with consent, for personalized results through trauma informed outcomes; polite and non-offensive language is used, and accessability (Braille, ASRS, visuals/tools for PWD, neurodiverse, etc.) is incorporated.</ul></li><ul><li>All relevant project records are minimal, aligns with record management policies, continiously follows all destruction proceedures and Records Office becomes engaged for uncommon concerns.
+5 | Adaptation | <ul><li>Product/Service working group embodies BIPOC, LGBTQIA2S+, Neurodiverse, People with Disabilities, Rural Residents, Socio-economic diverse, and Gender Diverse Folx.</ul></li><ul><li>Demographic (race, gender, immigration), health and various sensitive data is actively collected and used in aggravated form & with consent for the collective community while protecting human rights of the individual; outcomes and/or solutions of data mobilization are trauma informed. </ul></li><ul><li>Feedback system incorporates Equity Impact Assessment(s), empathy, and confidentiality, inclusive of open-ended questions & space for honesty without retaliation, occurs frequently for continious improvement, is delivered to working group immediately and efficiently for action. Follow-up with client includes accountability and humility.</ul></li><ul><li>Polite and non-offensive language is used, and accessability (Braille, ASRS, visuals/tools for PWD, neurodiverse, etc.) is incorporated.</ul></li><ul><li>All relevant project records are minimal, aligns with record management policies, continiously follows all destruction proceedures and Records Office becomes engaged for uncommon concerns.
```

Comment:
Demographic (race, gender, immigration), health and various sensitive data is actively collected and used in aggravated form & with consent for the collective community while protecting human rights of the individual; outcomes and/or solutions of data mobilization are trauma informed.
=> Demographic, health and various other sensitive data is actively collected with consent.
=> Aggregate data is consumed with a mindset to protecting human rights.
=> Solutions and outcomes are trauma informed.

Reply:
Feedback system incorporates Equity Impact Assessment(s), empathy, and confidentiality, inclusive of open-ended questions & space for honesty without retaliation, occurs frequently for continious improvement, is delivered to working group immediately and efficiently for action. Follow-up with client includes accountability and humility.

- I'm having a hard time understanding this one.  Chop down and/or break up.

Reply:
All relevant project records are minimal, aligns with record management policies, continiously follows all destruction proceedures and Records Office becomes engaged for uncommon concerns.

- continiously => continuously
- proceedures => procedures
- Make this two points.  The second can be about record destruction.

Reply:
Adjustments for clarity made.